internal/dag/ingress_processor.go

```diff
@@ -140,7 +140,15 @@ func (p *IngressProcessor) computeIngressRule(ing *v1beta1.Ingress, rule v1beta1
 			continue
 		}
 
-		r := route(ing, path, s, clientCertSecret, p.FieldLogger)
+		r, err := route(ing, path, s, clientCertSecret, p.FieldLogger)
+		if err != nil {
+			p.WithError(err).
+				WithField("name", ing.GetName()).
+				WithField("namespace", ing.GetNamespace()).
+				WithField("regex", path).
+				Errorf("path regex is not valid: %s", err)
```

Comment:
```suggestion
			p.WithError(err).
  				WithField("name", ing.GetName()).
  				WithField("namespace", ing.GetNamespace()).
  				WithField("regex", path).
  				Error("invalid path regular expression")
```

since the error is already logged by `WithError()`

Reply:
👍